showcase/src/assets/examples/collapsible/collapsible-custom2.css

```diff
@@ -0,0 +1,8 @@
+.collapsible-demo-button {
```

Comment:
Can you change the name of these files so it is clear what example they are related to? `collapsible-custom2` is too ambiguous...